climpyrical/gridding.py

```diff
@@ -53,40 +55,45 @@ def check_input_coords(x, y):
     check_ndims(x, 1)
     check_ndims(y, 1)
 
-    if (not np.isclose(x.max(), 33.8800048828125)) or (
-        not np.isclose(x.min(), -33.8800048828125)
+    if (not np.isclose(x.max(), ds.rlon.max())) or (
+        not np.isclose(x.min(), ds.rlon.min())
     ):
         raise ValueError(
             "x dimension array must have min/max values between \
-            -33.8800048828125 and 33.8800048828125. Array \
+            {} and {}. Array \
             provided has values between {} \
             and {}".format(
-                x.min(), x.max()
-            )
+                        ds.rlon.min(),
+                        ds.rlon.max(),
+                        x.min(),
+                        x.max()
+                    )
         )
 
-    if (not np.isclose(y.max(), 28.15999984741211)) or (
-        not np.isclose(y.min(), -28.59999656677246)
+    if (not np.isclose(y.max(), ds.rlat.max())) or (
+        not np.isclose(y.min(), ds.rlat.min())
     ):
         raise ValueError(
             "y dimension array must have min/max values between \
-            -28.59999656677246 and 28.15999984741211. Array \
+            {} and {}. Array \
             provided has values between {} \
             and {}".format(
-                y.min(), y.max()
+                ds.rlat.min(), ds.rlat.max(), y.min(), y.max()
             )
         )
```

Comment:
Given the identical pattern with these two checks, I'd suggest factoring them out to a single common function, called twice. DRY.